eZ/Bundle/EzPublishRestBundle/Resources/config/value_object_visitors.yml

```diff
@@ -196,6 +197,14 @@ services:
             - { name: ezpublish_rest.output.value_object_visitor, type: eZ\Publish\Core\REST\Server\Exceptions\BadRequestException }
             - { name: ezpublish_rest.output.value_object_visitor, type: eZ\Publish\Core\REST\Common\Exceptions\Parser }
 
+    ezpublish_rest.output.value_object_visitor.ContentFieldValidationException:
+        parent: ezpublish_rest.output.value_object_visitor.BadRequestException
+        class: %ezpublish_rest.output.value_object_visitor.ContentFieldValidationException.class%
+        arguments: [ %kernel.debug%, @translator ]
```

Comment:
@glye: Let's use double quotes around `"%parameters%"` and `"@service_ids"`. Referencing them without quotes is depreciated in Yaml (see #1744). You probably need to rebase against master and resolve possible conflicts, since #1744 was merged today. Sorry for the trouble :)